server.py

```
@@ -0,0 +1,57 @@
+        print(f"🚀 PromptPilot Web Preview Server")
+        print(f"📍 URL: {url}")
+        print(f"🌐 Server running on port {PORT}")
+        print(f"📱 Open your browser to view the app preview")
+        print(f"⏹️  Press Ctrl+C to stop the server")
+        print()
+        
+        # Try to open the browser automatically
+        try:
+            webbrowser.open(url)
+            print("✅ Browser opened automatically!")
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)